requirements-dev.txt

```diff
@@ -1,12 +0,0 @@
-flake8==3.3.0
```

Comment:
We use `pyup` bot to track dependencies updates like here https://github.com/aio-libs/aiohttp-jinja2/pull/129 Could you please check if `pyup` bot work with setup.py too. Otherwise lets leave as it is.